torch_disc/torch_disc/csrc/disc_compiler/passes/cluster.cpp

```diff
@@ -27,6 +27,7 @@ std::vector<Node*> FakeCluster(const std::shared_ptr<Graph>& graph) {
   for (auto node : graph->nodes()) {
     if (torch::blade::IsMlirMhloSupported(*node) &&
         node->kind() != prim::Constant) {
+      if (node->kind() == aten::addmm) continue;
```

Comment:
Is the aten::addmm skipped because it running into accuracy error?

Reply:
no, `admm` caused unchanged loss, it is a bug, will fix it later https://github.com/alibaba/BladeDISC/issues/215